ansible_testing/modules.py

```diff
@@ -181,7 +181,11 @@ def _just_docs(self):
             return False
 
     def _is_new_module(self):
-        return not module_loader.has_plugin(self.name)
+        if self.name.startswith("_"):
```

Comment:
We also need to ensure that the file is not a symlink.  Check out the logic that is in place at https://github.com/ansible/ansible/blob/4ed88512e45112f9670560ac3f01707a40a7f5c5/lib/ansible/plugins/__init__.py#L295-L306